services/meeting_service.js

```diff
@@ -192,6 +193,7 @@ const meetingService = {
                     .map(async slot => await TimeSlotsUtils.sliceSlots(slot, { startTime, duration }, host.uuid))
             )
         }
+        RabbitService.sendPickedMeetingDate(uuid, startTime, duration)
```

Comment:
To na pewno nie jest asynchroniczne?

Reply:
Tak samo jak powyżej XD znaczy mógłbym to zrobić asynchronicznie, ale nie wiem czy coś to pomoże skoro wewnątrz jest wołany setTimeout

Reply:
W sensie ok. Na wszelki wypadek pytam XDD

Reply:
Znaczy odpowiedzią pewnie może być to że mi testy nie przechodzą na githubie XDD